Source/JavaScriptCore/runtime/StackAlignment.h

```diff
@@ -39,6 +39,15 @@ constexpr unsigned stackAlignmentRegisters()
     return stackAlignmentBytes() / sizeof(EncodedJSValue);
 }
 
+// The number of bytes the SP needs to be adjusted downwards to get an aligned SP after a function prologue.
+// I.e.: (callFrameRegister - stackAdjustmentForAlignment()) % stackAlignmentBytes() == 0 always;
+constexpr unsigned stackAdjustmentForAlignment()
```

Comment:
WTF::roundUpToMultipleOf?

Reply:
actually, `roundUpToMultipleOf` isn't  `constexpr`--and this is a useful constant to have around (as a minimum frame size); not certain it's going to make a difference since I'd expect this function to actually be folded away